shelvery_tests/docdb_integration_test.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import traceback
+import unittest
+import pytest
+import yaml
+
+import boto3
+import os
+import time
+import botocore
+from datetime import datetime
+
+from shelvery_tests.test_functions import addBackupTags, clusterCleanupBackups, clusterShareBackups, compareBackups, initCleanup, initCreateBackups, initSetup, initShareBackups
+
+pwd = os.path.dirname(os.path.abspath(__file__))
+
+sys.path.append(f"{pwd}/..")
+sys.path.append(f"{pwd}/../shelvery")
+sys.path.append(f"{pwd}/shelvery")
+sys.path.append(f"{pwd}/lib")
+sys.path.append(f"{pwd}/../lib")
+
+from shelvery.documentdb_backup import ShelveryDocumentDbBackup
+from shelvery.engine import ShelveryEngine
+from shelvery.engine import S3_DATA_PREFIX
+from shelvery.runtime_config import RuntimeConfig
+from shelvery.backup_resource import BackupResource
+from shelvery.aws_helper import AwsHelper
+from shelvery_tests.conftest import destination_account
+
+print(f"Python lib path:\n{sys.path}")
+
+
+class ShelveryDocDBIntegrationTestCase(unittest.TestCase):
+    """Shelvery DocDB Backups Integration shelvery tests"""
+
+    def id(self):
+        return str(self.__class__)
+
+    def setUp(self):
+        self.created_snapshots = []
+        self.regional_snapshots = {
+            'ap-southeast-1': [],
+            'ap-southeast-2': []
+        }
+        os.environ['SHELVERY_MONO_THREAD'] = '1'
+
+        # Complete initial setup and create service client
+        initSetup(self,'docdb')
+        docdbclient = AwsHelper.boto3_client('docdb', region_name='ap-southeast-2')
+
+        #Wait till db is ready
+        status = docdbclient.describe_db_clusters(
+            DBClusterIdentifier='shelvery-test-docdb'
+        )['DBClusters'][0]['Status']
+
+        while str(status).lower() != 'available':
```

Comment:
in built waiter https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Waiter.DBClusterAvailable